zerver/models.py

```diff
@@ -214,8 +215,8 @@ def remote_user_to_email(remote_user):
 
 class RealmEmoji(models.Model):
     realm = models.ForeignKey(Realm)
-    name = models.TextField()
-    img_url = models.TextField()
+    name = models.TextField(validators=[MinLengthValidator(1), RegexValidator(regex=r'^[0-9a-zA-Z_\-\.]*$')])
+    img_url = models.URLField()
```

Comment:
Well, on the database the field is changed from `text` to `character varying(200)` as URLField enforces a `max_length=200` by default. Other than that there should be no problems as values are still retained, even if they don't represent valid URLs.